### ISSUE TITLE
'wallpaperOverlaySetting is undefined'

### ISSUE THREAD

**Issue by ForteDexe** [rishuinfinity/WallpaperSwitcher#8]
Happen frequently, logout and login just temporary solution.
Or alt+f2 -> r

**Comment by rishuinfinity** [comment]
Ok, working on fix, would release soon

**Comment by rishuinfinity** [comment]
Hey, I've uploaded the modified version on github. I wasn't able to replicate the error on my system, so would you be willing to install the github version of the extension and see if the problem persists in the near future. If error doesn't occur again then I would upload it on gnome-extensions website.

To install git version: 
```
git clone https://github.com/rishuinfinity/WallpaperSwitcher.git
cd WallpaperSwitcher
./install.sh
```
Thanks

**Comment by ForteDexe** [comment]
Ok, I will test it, thank you

**Comment by ForteDexe** [comment]
The issue still exists.
**wallpaperOverlaySetting is null**

**Comment by rishuinfinity** [comment]
I've changed the code at github again. Still I haven't been able to replicate the issue on my fedora system. Would you be kind enough to test the github version again? Thanks

**Comment by rishuinfinity** [comment]
Haash, Finally fixed in #11 
Can't believe the actual problem was missing brackets after if statement. :rofl:  

I am confident with this solution, I am also putting an update request on the extension website. The github version has been updated, If you wanna test out the github version. I am keeping this issue chat open for some time in case you find the bug still exists in your case.

**Comment by ForteDexe** [comment]
Haha, it happens to me sometimes, and makes me wonder where did I wrong.
The issue is fixed. thank you very much ^^.
